napari/_qt/range_slider/tests/range_slider_tests.py

```diff
@@ -0,0 +1,31 @@
+"""
+ Interactive test for range slider.
+"""
+
+# starts the QT event loop
+from napari._qt.range_slider.range_slider import QVRangeSlider
+from napari.util import app_context
+
+with app_context():
+
+    # creates a slider range widget::
+    widget = QVRangeSlider(slider_range=[0, 100, 1],
+                           values=[0, 50],
+                           parent=None)
+
+    # notify of changes while sliding:
+    widget.setEmitWhileMoving(True)
+
```

Comment:
can this stuff be moved to a separate PR?

Reply:
i think the newer range slider stuff is needed for some of the dims stuff to work properly, but unclear to me. i'd say we just leave it, including the tests